### ISSUE TITLE
Drag and drop causes page to scroll on iOS 11.3

### ISSUE THREAD

**Issue by JWA111** [taye/interact.js#631]
Opening this issue related to https://github.com/taye/interact.js/issues/561 

The page scrolls when the user drags an element on iOS 11.3. Please refer to the demos on [interactjs.io](interactjs.io) for a reproduction. 

### Expected behavior
When touch-dragging the page does not scroll

### Actual behavior
When touch-dragging the page scrolls

### System configuration
**interact.js version**: 1.3.3
**Browser name and version**: Safari iOS 11.3, Chrome 66.0.3359.122 on iOS 11.3
**Operating System**: iOS 11.3

Tested this issue on a physical iPad (latest gen)

**Comment by mcdemarco** [comment]
[This webkit issue](https://bugs.webkit.org/show_bug.cgi?id=184250) includes links to some other drag-and-drop libraries' issues, which may be helpful.

**Comment by JWA111** [comment]
For anyone who may be having this issue, here are three ways other libraries have worked around it:

https://github.com/Shopify/draggable/pull/200
https://github.com/atlassian/react-beautiful-dnd/pull/416
https://github.com/lazd/iNoBounce/pull/47

**Comment by taye** [comment]
Can you try using v1.4.0-alpha.5 (`npm install interactjs@next`) and calling the code below before you create any interactables?

```js
interact.addDocument(window.document, {
  events: { passive: false },
});
```

**Comment by taye** [comment]
`v1.3.4` has been published with a fix for the new iOS 11.3 issue. Thanks for the help.

**Comment by hilmanski** [comment]
Anyone faced this issue currently and know how to solve this? I dont's see passive attribute in docs for events
